lib/web/integrations_awsoidc.go

```diff
@@ -440,3 +440,61 @@ func (h *Handler) awsOIDCDeployEC2ICE(w http.ResponseWriter, r *http.Request, p
 		Name: resp.Name,
 	}, nil
 }
+
+// awsOIDCConfigureIdP returns a script that configures the AWS OIDC Integration in AWS.
+// This creates an OIDC Identity Provider that trusts this Teleport instance.
```

Comment:
```suggestion
```
We can remove this if you decide to take changes suggested above. "This" read a bit ambiguous to me.